zk-tests/test.sh

```diff
@@ -130,6 +154,34 @@ echo "Running script..."
 start_era_test_node
 RUST_LOG=warn "${BINARY_PATH}" script ./script/Deploy.s.sol:DeployScript --broadcast --private-key "0x3d3cbc973389cb26f657686445bcc75662b415b656078503592ac8c1abb8810e" --chain 260 --gas-estimate-multiplier 310 --rpc-url http://localhost:8011 --use "./${SOLC}" --slow  -vvv  || fail "forge script failed"
 RUST_LOG=warn "${BINARY_PATH}" script ./script/Deploy.s.sol:DeployScript --broadcast --private-key "0x3d3cbc973389cb26f657686445bcc75662b415b656078503592ac8c1abb8810e" --chain 260 --gas-estimate-multiplier 310 --rpc-url http://localhost:8011 --use "./${SOLC}" --slow  -vvv  || fail "forge script failed on 2nd deploy"
+echo "Running NFT script"
+RUST_LOG=warn "${BINARY_PATH}" script ./script/NFT.s.sol:MyScript --broadcast --private-key $PRIVATE_KEY --rpc-url $RPC_URL --use 0.8.20 --zksync  || fail "forge script failed"
+
+# Deploy ERC20
+echo "Deploying MyToken..."
+MYTOKEN_DEPLOYMENT=$(RUST_LOG=warn "${BINARY_PATH}" create ./src/ERC20.sol:MyToken --rpc-url $RPC_URL --private-key $PRIVATE_KEY --use 0.8.20 --zksync) || fail "forge script failed"
+MYTOKEN_ADDRESS=$(echo $MYTOKEN_DEPLOYMENT | grep -oE '0x[a-fA-F0-9]{40}')
```

Comment:
This regex will match all hexadecimal strings in the output. Maybe we can be explicit what we are matching for

```bash
$(echo $MYTOKEN_DEPLOYMENT) | grep -oP '(?<=Deployed to: )0x\w{40}'
```

But I think OSX ships with the BSD grep so "-P" won't work and neither would the lookbehind, so might have to go with 

```bash
$(echo $MYTOKEN_DEPLOYMENT) | grep -E 'Deployed to: ' | grep -oE '0x\w+'
```